test/tests/functional/pbs_qsub_remove_files.py

```diff
@@ -49,11 +49,130 @@ def test_remove_file_when_job_succeeds(self):
         submit a sleep job and make sure that the std_files
         are getting deleted when remove_files option is used.
         """
+        j = Job(TEST_USER, attrs={ATTR_R: 'oe'})
+        j.set_sleep_time(1)
+        sub_dir = DshUtils().mkdtemp(uid=TEST_USER.uid)
+        jid = self.server.submit(j, submit_dir=sub_dir)
+        self.server.expect(JOB, {'Remove_Files': 'oe'}, id=jid)
+        self.server.log_match(jid + ";dequeuing from", max_attempts=10)
```

Comment:
can't we make sure that the job has finished instead of relying on this message?

Reply:
Hi @arungrover , Do you want to me make history enabled and check for F state ?

Reply:
That can work too, or just check for non-existence of job and then proceed